packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -153,6 +155,7 @@ class DateAbstract extends React.Component {
 
         >
           <StyledDateAbstractInput
+            disabled={this.props.immutable && 'disabled'}
```

Comment:
`{...this.props.immutable ? {disabled: 'disabled} : {}}`

i think like this would be a little bit better. Since its more clear imo and omits the disabled attribute completely if not immutable.